ui/src/components/SituationCard.vue

```diff
@@ -2,67 +2,73 @@
 import SeverityStatus from '@/elements/SeverityStatus.vue'
 import AlarmsCountBySeverity from '@/components/AlarmsCountBySeverity.vue'
 import { TSituation } from '@/types/TSituation'
-
 const props = defineProps<{
 	alarmInfo: TSituation
+	selected: boolean
 }>()
-
 const emit = defineEmits(['situation-selected'])
 const handleSituationSelected = () => {
 	emit('situation-selected', props.alarmInfo?.id)
 }
 </script>
 
 <template>
-	<div v-on:click="handleSituationSelected" class="card">
+	<div
+		v-on:click="handleSituationSelected"
+		class="card"
+		v-bind:class="{ selected: props.selected }"
```

Comment:
not a big deal but mix of both `v-bind:class` here and just `:class` below, which do the same